dev-libs/angelscript/angelscript-2.35.1.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A flexible cross-platform scripting library"
+DESCRIPTION="The AngelCode Scripting Library, or AngelScript as it is also known, \
+is an extremely flexible cross-platform scripting library designed to allow applications to \
+extend their functionality through external scripts. It has been designed from the beginning \
+to be an easy to use component, both for the application programmer and the script writer"
+HOMEPAGE="http://www.angelcode.com/angelscript/"
+COPYRIGHT="2003-2021 Andreas Jönsson"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="http://www.angelcode.com/angelscript/sdk/files/angelscript_2.35.1.zip"
+CHECKSUM_SHA256="5c1096b6d6cf50c7e77ae93c736d35b69b07b1e5047161c7816bca25b413a18b"
+SOURCE_DIR="sdk"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	angelscript$secondaryArchSuffix
+	lib:angelscript$secondaryArchSuffix
+	lib:libangelscript$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	angelscript${secondaryArchSuffix}_devel
+	devel:angelscript
+	devel:libangelscript
+	"
+REQUIRES_devel="
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:cc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD() {
+	cd angelscript/projects/cmake
+	cmake . -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_INSTALL_PREFIX=$prefix \
+		-DCMAKE_INSTALL_INCLUDEDIR="$includeDir" -DBUILD_SHARED_LIBS=ON
+	make $jobArgs
+}
+
+INSTALL() {
+	cd angelscript/projects/cmake
+	make install
+	mkdir -p $includeDir
+	mv $prefix/include/* $includeDir
+	rmdir $prefix/include/
```

Comment:
Would be nice to get a small patch to the CMakeLists.txt file so that it actually respects the `CMAKE_INSTALL_INCLUDEDIR` variable you're passing in the BUILD section.  Shouldn't be too hard but I don't think it's required for this to be merged.